mscore/musescore.cpp

```diff
@@ -4599,6 +4599,26 @@ void MuseScore::cmd(QAction* a)
                QString("Command %1 not valid in current state").arg(cmdn));
             return;
             }
+      if (cmdn == "palette-search") {
+            PaletteBox* pb = getPaletteBox();
+            QLineEdit* sb = pb->searchBox;
+            sb->setFocus();
+            if (pb->noSelection())
```

Comment:
Are you aware that we will not pass here if the searchbox is used by just clicking on it and entering text? keyboardNavigation will not be changed then, is that a problem?